dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -105,10 +106,21 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     }
     CATCH_RETURN()
 
+    bool PushNotificationManager::IsSupported()
+    {
+        // Only scenarios that use the Background Infrastructure component of the OS support Push in the SelfContained case
+        if (WindowsAppRuntime::SelfContained::IsSelfContained() && !PushNotificationHelpers::IsPackagedAppScenario())
```

Comment:
Suggest:
`static bool isSelfContained { WindowsAppRuntime::SelfContained::IsSelfContained() && !PushNotificationHelpers::IsPackagedAppScenario() }`

So we only calculate this one time.

Reply:
done